### PR TITLE
Add debug tools classes

### PR DESCRIPTION
**Pull request by corsonr**
Pretty handy in particular if some buttons require a javascript based action (ex: "are you sure you want to delete this?")

### CODE REVIEW

File: includes/admin/views/html-admin-page-status-tools.php [hunk 1]
```diff
@@ -20,11 +20,11 @@
 		</thead>
 		<tbody class="tools">
 			<?php foreach ( $tools as $action => $tool ) : ?>
-				<tr>
+				<tr class="<?php echo $action; ?>">
```
Comment (justinshreve) [inline includes/admin/views/html-admin-page-status-tools.php]: This should be wrapped in [esc_attr](https://codex.wordpress.org/Function_Reference/esc_attr).

File: includes/admin/views/html-admin-page-status-tools.php [hunk 2]
```diff
@@ -20,11 +20,11 @@
...
 					<td><?php echo esc_html( $tool['name'] ); ?></td>
 					<td>
 						<p>
-							<a href="<?php echo wp_nonce_url( admin_url('admin.php?page=wc-status&tab=tools&action=' . $action ), 'debug_action' ); ?>" class="button"><?php echo esc_html( $tool['button'] ); ?></a>
+							<a href="<?php echo wp_nonce_url( admin_url('admin.php?page=wc-status&tab=tools&action=' . $action ), 'debug_action' ); ?>" class="button <?php echo $action; ?>"><?php echo esc_html( $tool['button'] ); ?></a>
```
Comment (justinshreve) [inline includes/admin/views/html-admin-page-status-tools.php]: So should the `$action` here.

**Comment by corsonr** [comment]
what do you think @claudiosmweb ?

**Comment by claudiosmweb** [comment]
@corsonr I liked this. But yes, `sanitize_html_class` will make more sense for it.

**Comment by corsonr** [comment]
fixed @claudiosmweb

**Comment by claudiosmweb** [comment]
@corsonr <3

**Pull request merged**